src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,209 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-toggle-switch {
+@mixin iui-toggle-switch-wrapper {
   @include iui-reset;
   display: flex;
   align-items: center;
+  gap: $iui-s;
+  align-items: center;
   font-size: $iui-font-size;
   width: fit-content;
-  padding: ($iui-baseline/2) 0;
   user-select: none;
   cursor: pointer;
-  position: relative;
+  isolation: isolate;
   @include themed {
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
+  // Disabled
+  &.iui-disabled {
+    cursor: not-allowed;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  // IE solution
+  // For labels on the left
+  > .iui-label {
+    margin-right: $iui-s;
+    @supports (gap: $iui-s) {
+      margin-right: 0;
+    }
+  }
+
+  // For labels on the right
+  .iui-toggle-switch ~ .iui-label {
+    margin-left: $iui-s;
+    margin-right: 0;
+    @supports (gap: $iui-s) {
+      margin-left: 0;
+    }
+  }
+}
+
+@mixin iui-toggle-switch {
+  --_iui-toggle-switch-handle-size: #{$iui-icons-default};
```

Comment:
What's the point having these as CSS variables instead of SASS variables? We are not overrding them with any of modifier. Also, now you have to have fallbacks for IE.

Reply:
Lines 66-67 are still using SASS variables, so I believe it should still display ok in IE.  The CSS variables allowed me to test the scaling of the components, but I can revert them back to SASS.  I thought we were starting to switch over to CSS variables tho?

Reply:
I don't really see a point to have them as CSS variables as we are not changing them dynamically. And I think we were talking about switching to CSS variables when we will end IE support and we will have separate lib for varaibles 😀

Reply:
Ok, I have removed the CSS variables.